src/rules/russian.toml

```diff
@@ -0,0 +1,78 @@
+min_trimmed_length = 3
+
+# For count = 2 we have sentences like: "Это он", "Будет беда"
+# and a lot of trash abbreviations like "до н.", "и доп."
+min_word_count = 3
+
+max_word_count = 14
+
+# In Russian, words can consist of one letter, so no restrictions
+min_characters = 0
+
+may_end_with_colon = false
+quote_start_with_letter = true
+needs_punctuation_end = false
+needs_letter_start = true
+
+# Apparently, in some places the sentences are cut incorrectly,
+# which is why we get some part of the sentence, and not its entirety.
+# This is required to fix sentences like:
+# с Иваном Галамяном.
+needs_uppercase_start = true
+
+allowed_symbols_regex = "[А-Яа-яёЁ\\s:,.\\-‑?;!—­‐–'·=’―−”‘]"
```

Comment:
Note that your previous version excluded all quotes and some other characters as it lead to uneven symbol occurances in the sentences. This now allows quotes, so you might end up with sentences with wrong symbols. I'd suggest to add these to the `even_symbols` config option to make sure quotes (and possibly other symbols) are always even.

Reply:
The may difference with the previous version, is that I allowed `‘` symbol - that is ok, in Russian there can be words like "Côte d'Ivoire".

Please, can you provide more info about `even_symbols`, I can't find detailed description about it.

Reply:
With this regex here, you could potentially get the following sentence: `This is ”uneven and suddenly stops the quote without terminating symbol`

Regarding `even_symbols`, can you tell me what exactly is not clear from the README? Then we can adjust the README to provide better info

Reply:
Did you have a chance to look at this? Is there anything I can help with?